openpype/hosts/houdini/plugins/publish/validate_vdb_output_node.py

```diff
@@ -56,12 +89,22 @@ def get_invalid(cls, instance):
         nr_of_prims = len(prims)
 
         # All primitives must be hou.VDB
-        invalid_prim = False
+        invalid_prims = []
```

Comment:
maybe we need to introduce to check whether the node is VDB type. 
Consider the pseudo code below 
```
        main_node = hou.node(instance.data["instance_node"])
        vdb_path = main_node.evalParm("soppath")
        vdb = hou.node(vdb_path)
        if not isinstance(vdb, hou.VDB):
            cls.log.error("the output object is not VDB type")
            return [node]
```

Reply:
Isn't that exactly what it's doing by checking whether the primitives are VDB types?
There isn't a "VDB" node type right? There's only the `hou.VDB` geometry type which is exactly what we're checking here.

However, your case of a "grid" passing just fine shouldn't happen so I'll investigate.